srml/fees/src/lib.rs

```diff
@@ -14,7 +14,114 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Handles all transaction fee related operations
+//! ## Overview
+//! 
+//! The `fees` module handles transaction fee related operations. To use this module, the [`TransferAsset`](https://crates.parity.io/srml_fees/trait.Trait.html#associatedtype.TransferAsset) trait must be implemented to enable asset transfer between accounts. This module provides the following functionality
+//! 
+//! - Getting and setting the base and per-byte transaction fee
+//! - Tracking the overall charged fees for each transaction
+//! - Charging and refunding transaction fees
+//! 
+//! ## Interface
+//! 
+//! ### Dispatchable
+//! 
+//! This module is intended to be called by other modules rather than transactions (no dispatchable functions).
+//! 
+//! ### Public
+//! 
+//! The following methods are publicly accessible to other modules that implement the [`ChargeFee`](https://github.com/paritytech/substrate/blob/master/srml/fees/src/lib.rs#L93) trait.
+//! 
+//! - [`charge_fee`](https://github.com/paritytech/substrate/blob/master/srml/fees/src/lib.rs#L96) - handles subtracting transaction fees from the transactor's account
+//! - [`refund_fee`](https://github.com/paritytech/substrate/blob/master/srml/fees/src/lib.rs#L107) - handles transaction fee refunds to the transactor's account
+//! 
+//! ## Usage
+//! 
+//! ### Prerequisites
+//! 
+//! The fees module depends on the [`system`](https://crates.parity.io/srml_system/index.html) and [`support`](https://crates.parity.io/srml_support/index.html) modules as well as Substrate Core libraries and the Rust standard library.
+//! 
+//! ### Simple Code Snippet
+//! 
+//! Use the `ChargeFee` trait to charge and refund fees.
+//! 
+//! ```rust,ignore
+//! T::ChargeFee::charge_fee(transactor, fee);
+//! T::ChargeFee::refund_fee(transactor, fee);
+//! ```
+//! 
+//! ### Example from SRML
+//! 
+//! The `ChargeFee` trait is invoked in the `balances` and `contract` modules to manage fee-related logic in the following ways
```

Comment:
it is not anymore in balances no ? it only has fees for creation and transfer but not related to transaction fee of this module as far as I understand